qutebrowser/completion/models/miscmodels.py

```diff
@@ -48,8 +51,18 @@ def __init__(self, parent=None):
         for name, cmd in config.section('aliases').items():
             cmdlist.append((name, "Alias for '{}'".format(cmd)))
         cat = self.new_category("Commands")
+
+        # map each command to its bound keys and show these in the misc column
+        keyconf = objreg.get('key-config')
+        cmd_to_keys = defaultdict(list)
+        for key, cmd in keyconf.get_bindings_for('normal').items():
+            # put special bindings last
+            if key.startswith('<') and key.endswith('>'):
+                cmd_to_keys[cmd].append(key)
+            else:
+                cmd_to_keys[cmd].insert(0, key)
         for (name, desc) in sorted(cmdlist):
-            self.new_item(cat, name, desc)
+            self.new_item(cat, name, desc, str.join(', ', cmd_to_keys[name]))
```

Comment:
Didn't notice this with the first review - this should be `', '.join(cmd_to_keys[name])` (as `join` is a method on strings, and commonly used like that).